src/particles/ReferenceParticle.H

```diff
@@ -19,6 +19,8 @@
 #include <AMReX_REAL.H>
 #include <AMReX_SmallMatrix.H>
 
+#include <CompensatedReal.H>
```

Comment:
Since this is an ImpactX file, it should be the first import and
```suggestion
#include "CompensatedReal.H"
```